From e32c4b8757278bbc8ef6b23af1ed414cb21d1cd1 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Tue, 23 May 2023 14:43:41 +0530
Subject: [PATCH] Fixed Sepolicy Build Issue.

There were hash diff issue in sepolicy. Updated
missing policy in files properly.

Tracked-On: OAM-106853
Change-Id: I428d85da2db21401941324be09bf71c5e47a7176
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 prebuilts/api/34.0/private/compat/29.0/29.0.ignore.cil | 1 +
 prebuilts/api/34.0/public/hal_neverallows.te           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/prebuilts/api/34.0/private/compat/29.0/29.0.ignore.cil b/prebuilts/api/34.0/private/compat/29.0/29.0.ignore.cil
index e40888dcc..1cb94021a 100644
--- a/prebuilts/api/34.0/private/compat/29.0/29.0.ignore.cil
+++ b/prebuilts/api/34.0/private/compat/29.0/29.0.ignore.cil
@@ -88,6 +88,7 @@
     ota_metadata_file
     ota_prop
     prereboot_data_file
+    proc_locks
     art_apex_dir
     rebootescrow_hal_prop
     securityfs
diff --git a/prebuilts/api/34.0/public/hal_neverallows.te b/prebuilts/api/34.0/public/hal_neverallows.te
index e77ea9df0..e6c90756d 100644
--- a/prebuilts/api/34.0/public/hal_neverallows.te
+++ b/prebuilts/api/34.0/public/hal_neverallows.te
@@ -29,6 +29,7 @@ neverallow {
   -hal_wifi_hostapd_server
   -hal_wifi_supplicant_server
   -hal_telephony_server
+  -hal_sensors_server
   -hal_uwb_server
   # TODO(b/196225233): Remove hal_uwb_vendor_server
   -hal_uwb_vendor_server
-- 
2.17.1

